other-docs/guides/upgrading/v23.md

```diff
@@ -0,0 +1,115 @@
+---
+order: 23
+---
+
+# Upgrading to v23
+
+*If you are migrating from WordPress to Altis, check out the [migrating guide](../migrating/) first.*
+
+To upgrade to Altis v23, edit your `composer.json` and change the version constraint for `altis/altis` and any local environment
+modules to `^23.0.0`.
+
+```json
+{
+	"require": {
+		"altis/altis": "^23.0.0"
+	},
+	"require-dev": {
+		"altis/local-server": "^23.0.0"
+	},
+	"config": {
+		"platform": {
+			"php": "8.2"
+		}
+	}
+}
+```
+
+Once you have made these changes, run `composer update` and then run the `wp altis migrate` command:
+
+```sh
+# For cloud environments
+wp altis migrate
+
+# For local server
+composer server cli -- altis migrate
+```
+
+## WordPress 6.8
+
+Altis adds support to WordPress 6.8, "Cecil", which brings significant improvements to performance, security, and usability:
+
+- **Security**: bcrypt is now used for password hashing, replacing phpass. Application keys use stronger BLAKE2b hashing. Admins are alerted if open registration is active with high default roles.
+
+- **Performance**: Improved caching in WP_Query, better lazy loading, and speculative loading via the new Speculation Rules API for faster navigation.
+
+- **Editor Enhancements**: Updated Style Book UI, new Query Loop options (date filters, exclude sticky posts), and persistent rendering mode for editing experience.
+
+- **Accessibility**: Over 100 fixes across the admin and editor, including standardized tooltips and clearer admin notice prefixes.
+
+For more details check the [WordPress 6.8 release announcement](https://wordpress.org/download/releases/6-8/).
```

Comment:
The field guide link https://make.wordpress.org/core/2025/03/28/wordpress-6-8-field-guide/ will be more approriate here as it gives more information about the new features.